serverscommands/instancecommands/common.go

```diff
@@ -33,11 +33,16 @@ func idOrName(c *cli.Context, client *gophercloud.ServiceClient) string {
 var idAndNameFlags = []cli.Flag{
 	cli.StringFlag{
 		Name:  "id",
-		Usage: "[optional; required if 'name' is not provided] The ID of the server to update",
+		Usage: "[optional; required if 'name' is not provided] The ID of the server to update.",
 	},
 	cli.StringFlag{
 		Name:  "name",
-		Usage: "[optional; required if 'id' is not provided] The name of the server to update",
+		Usage: "[optional; required if 'id' is not provided] The name of the server to updat.e",
+	},
+	cli.StringFlag{
+		Name:  "sep",
+		Usage: "[optional] The separator to use for splitting multiple IDs or names.",
+		Value: ",",
```

Comment:
This is a bit more than that though. This way, users can also pass in a comma-separated string of IDs, for example:
`rack servers instance delete --id id1,id2,id3 --sep ","`
I also think it's important to give users some leeway about how their data is formatted.